internal/util/importutil/import_wrapper.go

```diff
@@ -156,6 +187,14 @@ func (p *ImportWrapper) fileValidation(filePaths []string, rowBased bool) error
 // filePath and rowBased are from ImportTask
 // if onlyValidate is true, this process only do validation, no data generated, callFlushFunc will not be called
 func (p *ImportWrapper) Import(filePaths []string, rowBased bool, onlyValidate bool) error {
+	log.Info("ImportWrapper: filePaths", zap.Any("filePaths", filePaths))
+	// data restore function to import milvus native binlog files(for backup/restore tools)
+	// the backup/restore tool provide two paths for a collection, the first path is binlog path, the second is deltalog path
+	if p.isBinlogImport(filePaths) {
```

Comment:
So we are not calling `debug.FreeOSMemory()` in while importing binlogs? Could it be a problem when we have a lot of binlog files?